charts/sysdig/templates/secrets.yaml

```diff
@@ -14,9 +14,9 @@ data:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "sysdig.fullname" . }}
+  name: {{ .name }}
   labels:
-{{ include "sysdig.labels" . | indent 4 }}
+{{ include "sysdig.labels" $ | indent 4 }}
 type: Opaque
 data: {{ toYaml .data | nindent 2 }}
```

Comment:
I didn't get to comment on your previous pr, but now I have a chance :) 
I don't think we use nindent often, maybe its better if we keep it constant?